API.md

```diff
@@ -1,3 +1,57 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents** 
+
+- [Building Applications With Open MCT](#building-applications-with-open-mct)
+  - [Scope and purpose of this document](#scope-and-purpose-of-this-document)
+  - [Building From Source](#building-from-source)
+  - [Starting an Open MCT application](#starting-an-open-mct-application)
+  - [Plugins](#plugins)
+    - [Defining and Installing a New Plugin](#defining-and-installing-a-new-plugin)
+  - [Domain Objects and Identifiers](#domain-objects-and-identifiers)
+    - [Object Attributes](#object-attributes)
+    - [Domain Object Types](#domain-object-types)
+  - [Root Objects](#root-objects)
+  - [Object Providers](#object-providers)
+  - [Composition Providers](#composition-providers)
+    - [Adding Composition Providers](#adding-composition-providers)
+    - [Default Composition Provider](#default-composition-provider)
+  - [Telemetry API](#telemetry-api)
+    - [Integrating Telemetry Sources](#integrating-telemetry-sources)
+      - [Telemetry Metadata](#telemetry-metadata)
+        - [Values](#values)
+          - [Value Hints](#value-hints)
+        - [The Time Conductor and Telemetry](#the-time-conductor-and-telemetry)
+      - [Telemetry Providers](#telemetry-providers)
+      - [Telemetry Requests](#telemetry-requests)
+      - [Request Strategies **draft**](#request-strategies-draft)
+        - [`latest` request strategy](#latest-request-strategy)
+        - [`minmax` request strategy](#minmax-request-strategy)
+      - [Telemetry Formats **draft**](#telemetry-formats-draft)
+        - [Registering Formats](#registering-formats)
+      - [Telemetry Data](#telemetry-data)
+        - [Telemetry Datums](#telemetry-datums)
+      - [Limit Evaluators **draft**](#limit-evaluators-draft)
+    - [Telemetry Visualization APIs **draft**](#telemetry-visualization-apis-draft)
+  - [Time API](#time-api)
+    - [Time Systems and Bounds](#time-systems-and-bounds)
+      - [Defining and Registering Time Systems](#defining-and-registering-time-systems)
+      - [Getting and Setting the Active Time System](#getting-and-setting-the-active-time-system)
+    - [Time Bounds](#time-bounds)
+  - [Clocks](#clocks)
```

Comment:
Should be below Time API. Think it's due to incorrect number of #s in titles

Reply:
Fixing.